src/test/resources/checkstyle.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.2//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
+
+<module name="Checker">
+
+    <property name="charset" value="UTF-8"/>
+
+    <module name="TreeWalker">
+
+        <module name="OneStatementPerLine"/>
+
+    </module>
+
+</module>
```

Comment:
good work, can you please update checks with rules from https://github.com/wildfly/wildfly-checkstyle-config?